src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -555,6 +622,20 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
         }
     }
 
+    public static Map<String, String> getDefaultProperties(String filePath, String moduleName)
+            throws CheckstyleException {
+
+        final String fullyQualifiedClassName = getFullyQualifiedClassName(filePath, moduleName);
+        final ModuleDetails moduleDetails = MODULE_DETAILS_MAP.get(fullyQualifiedClassName);
+
+        final List<ModulePropertyDetails> properties = moduleDetails.getProperties();
+
+        final Map<String, String> result = new HashMap<>();
+        properties.forEach(prop -> result.put(prop.getName(), prop.getDefaultValue()));
```

Comment:
```suggestion
        final Map<String, String> result = new HashMap<>();
        moduleDetails.getProperties().forEach(prop -> result.put(prop.getName(), prop.getDefaultValue()));
```
field is only for readability, right?

Reply:
done.